src/components/button/Button.tsx

```diff
@@ -44,6 +44,7 @@ export const Button: FC<Props> = ({
   const component = (
     <RebassButton
       variant={intent}
+      as={href ? 'span' : 'button'}
```

Comment:
both a Button and a tag are focusable components so using the keyboard Tab would visit the same component twice, this is to remove button when we have a tag